src/controllers/v2/registration.js

```diff
@@ -3,9 +3,6 @@ import {sendSuccessEmail} from '../../notify-emails.js';
 
 const {Registration, Return, NonTargetSpecies, Revocation, Note, RequestUUID} = db;
 
-const REGISTRATION_RENEWAL = 'Renewal';
-const REGISTRATION_INITIAL = 'Initial';
```

Comment:
@RAnderson96 - Is this now set via the incoming API data? Can only find a get in this code...

Reply:
Yes thats right, its within the payload.